scripts/commit_classifier.py

```diff
@@ -109,9 +109,15 @@ def classify(self, diff_id):
 
         feature_names = self.model.get_human_readable_feature_names()
 
+        if len(probs[0]) > 2:
+            pred_class_index = probs.argmax(axis=-1)[0]
+        else:
+            pred_class_index = 0
+
+        pred_class = self.model.class_names[pred_class_index]
         features = []
         for i, (val, feature_index, is_positive) in enumerate(
-            importance["importances"]
+            importance["importances"]["classes"][pred_class][0]
```

Comment:
Here too, we know the class name as it's a binary classifier, I think you can just hardcode "1" instead of `pred_class`